<launch>
  <!-- multi_goal_loop -->
  <node pkg="dashgo_tools" name="auto_goal_path" type="auto_goal_path" output="screen">
    <rosparam param= "waypoint">
    <!--x, y, z, w-->
    [
     0.0,0.0,0.0,0.0,
     0.0,0.0,0.0,1.0,
     3.0,0.0,0.0,1.0,
     3.0,0.0,-0.707,0.707,
     3.0,-2.8,-0.707,0.707,
     3.0,-2.8,0.0,1.0,
     2.0,-2.8,0.0,1.0,
     3.0,-4.8,-0.707,0.707,
     3.0,-4.8,0.0,1.0,
     2.0,-4.8,0.0,1.0,
     3.0,-4.8,0.707,0.707,
     3.0,-2.8,0.707,0.707
    ]
</rosparam>

  </node>


  <node pkg="dashgo_tools" name="auto_path" type="auto_path" output="screen">

<!--start_node & end_node-->
    <!--param name="start_node" value="5" /-->
    <!--am name="end_node" value="7" /-->
<!--map param set-->
    <param name="nodenum" value="11" />
    <param name="linenum" value="14" />
    <rosparam param= "p">
    [
     0,1,2,3,4,4,5,5,6,7,8,8,8,9,10,11
    ]
</rosparam>
    <rosparam param= "q">
    [
     0,2,3,4,5,7,4,6,5,8,7,9,10,8,11,5
    ]
</rosparam>
    <rosparam param= "len">
    [
     0,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2
    ]
</rosparam>

    <rosparam param= "waypoint">
    <!--x, y, z, w-->
    [
     0.0,0.0,0.0,0.0,
     0.0,0.0,0.0,1.0,
     3.0,0.0,0.0,1.0,
     3.0,0.0,-0.707,0.707,
     3.0,-2.8,-0.707,0.707,
     3.0,-2.8,0.0,1.0,
     2.0,-2.8,0.0,1.0,
     3.0,-4.8,-0.707,0.707,
     3.0,-4.8,0.0,1.0,
     2.0,-4.8,0.0,1.0,
     3.0,-4.8,0.707,0.707,
     3.0,-2.8,0.707,0.707
    ]
</rosparam>
  </node>



</launch>
